demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -49,6 +61,7 @@ def set_commands(self):
             Command(
                 # the related to has to be a command
                 name=r.content_item.name,  # type: ignore[union-attr]
+                marketplaces=r.content_item.marketplaces,
```

Comment:
Why does a command need the `marketplaces`?
How does this work with common commands such as `reputation` commands that might have different versions (i.e. one with [XSOAR, v2], another with just [XSOAR] and a 3rd with just [v2]).

Shouldn't this be on the `edge`?

Reply:
The marketplace attribute of a command node is inherited of the implementations. For example, if `ip` command is used only by XSOAR integrations, the `marketplace` field will be `xsoar`. If it is used by both, we will receive ['xsoar', 'v2']. We can search for a command and the command model will provide us all the implementations of it.

In addition, we use the same command model for a specific command implementation, and that's why we have additional optional fields of `description` and `deprecated`. In this case, we should provide the `marketplaces` from the specific integration and not from the field, and I fixed it right now.